src/components/Bars/components/HorizontalBars.tsx

```diff
@@ -0,0 +1,58 @@
+import { ColorUtils } from "@/export";
+import { useGraph } from "@/hooks/use-graph/use-graph";
+import { CoordinatesUtils } from "@/utils/coordinates/coordinates";
+import { cx } from "@/utils/cx/cx";
+import { GraphUtils } from "@/utils/graph/graph";
+import React from "react";
+
+type Props = React.SVGAttributes<SVGSVGElement> & {
+	children?: React.ReactNode;
+};
+
+export const HorizontalBars = ({ children, className }: Props) => {
+	const context = useGraph();
+
+	if (!context) return null;
```

Comment:
No `if(!context)` check required anymore. Context is always there or it throws now under new system.

Reply:
done